pkg/api/jobs.go

```diff
@@ -104,9 +109,11 @@ func PrintJobsReport(w http.ResponseWriter, req *http.Request, currReport, twoDa
 
 	// If requesting a two day report, we make the comparison between the last
 	// period (typically 7 days) and the last two days.
+	// Otherwise the default of last 7 days vs last 14 days.
 	var current, previous []v1sippyprocessing.JobResult
-	switch req.URL.Query().Get("period") {
-	case "twoDay":
+	period := req.URL.Query().Get("period")
+	switch period {
+	case periodTwoDay:
 		current = twoDayPeriod
 		previous = currentPeriod
```

Comment:
I *think* I do this wrong in the new UI, since it compares 2 days to the previousPeriod, but I think both comparisons aren't ideal.  It might be nice to change the reporting to compare 2 days to the previous 7 days (e.g. 9 days ago to 2 days ago) once it's all in a DB.